code/__DEFINES/cult.dm

```diff
@@ -28,6 +28,7 @@
 #define SOULS_TO_REVIVE 3
 #define BLOODCULT_EYE "f00"
 //soulstone & construct themes
+#define THEME_CULT "cult" // actual cult, only use for cult constructs
 #define THEME_WIZARD "wizard"
-#define THEME_CULT "cult"
+#define THEME_FAKE_CULT "fake_cult"
```

Comment:
What's this? why?

Reply:
Wizard construct shards have a cult theme that can appear. However I wanted the new proc to handle that to also be able to handle actual cult constructs (not the "fake" cult themed constructs that aren't cult) hence this.